roles/grafana_agent/tasks/main.yml

```diff
@@ -0,0 +1,53 @@
+---
+- name: Include secrets
+  include_vars: "{{ secrets_path | mandatory }}/mimir_password.yml"
+  no_log: true
+  tags:
+    - always
+
+- name: "Import Grafana GPG key"
+  become: true
+  ansible.builtin.apt_key:
```

Comment:
I understand we've used `apt_key` for a while, but as [the `apt_key` docs explain](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/apt_key_module.html), the latest Debian/Ubuntu will no longer support this.

It's ok for today, but in the future, prepare to use the alternative example in the [`apt_repository` docs](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/apt_repository_module.html) (the `get_url` module plus the `signed-by` setting).

Reply:
We should do this the new way, and we should also work on fixing all the rest of the apt-key invocations.  One of the Debs already requires it and I'm not sure but I think newer Ubuntu may as well.

Reply:
> We should do this the new way, and we should also work on fixing all the rest of the apt-key invocations. One of the Debs already requires it and I'm not sure but I think newer Ubuntu may as well.

I updated the key import task to ansible.builtin.get_url and tested it on apt/rpm machines.